### PR TITLE
test: Add unit test for bug report on normalizing quantities

### PR DESCRIPTION
**Pull request by yuktea**
A test PR

### CODE REVIEW

File: t/food.t [hunk 1]
```diff
@@ -72,6 +72,9 @@ is( normalize_serving_size("42\N{U+516C}\N{U+5347}"), 42000 );
 is( unit_to_g(42, "\N{U+516C}\N{U+5347}"), 42000 );
 is( g_to_unit(42000, "\N{U+516C}\N{U+5347}"), 42 );
 
+#test for a bug
```
Comment (stephanegigandet) [inline t/food.t:75]: Maybe we could add the link to the actual bug.

**Comment by stephanegigandet** [comment]
2 unrelated files were deleted in this PR.

File: t/food.t (hunk 1 above)
Comment (yuktea) [reply t/food.t:75]: > 2 unrelated files were deleted in this PR.

@stephanegigandet This is strange. Not sure how that happened but I'll be more careful now. I don't remember staging these changes.

Comment (yuktea) [reply t/food.t]: Done and done

**Comment by yuktea** [comment]
> 2 unrelated files were deleted in this PR.

Reverted. Wasn't mindful enough while staging.

**Comment by stephanegigandet** [comment]
The test looks good, and it is failing which shows it's working. Once the bug is fixed, we can merge the test.

**Pull request closed without merging**